rclcpp/test/rclcpp/test_node.cpp

```diff
@@ -3346,6 +3346,9 @@ TEST_F(TestNode, static_and_dynamic_typing) {
     EXPECT_THROW(
       node->get_parameter("integer_override_not_given"),
       rclcpp::exceptions::ParameterUninitializedException);
+    EXPECT_THROW(
+      node->get_parameters({"integer_override_not_given"}),
+      rclcpp::exceptions::ParameterUninitializedException);
```

Comment:
this is just repeating L3346-3348

Reply:
Here is to test get_parameter**s**, L3346-3348 tests get_parameter.

Reply:
And this is what I reported the issue in https://github.com/ros2/rclcpp/issues/2038

Reply:
sorry, I didn't notice the difference